pykern/pkio.py

```diff
@@ -18,11 +18,26 @@
 import re
 import shutil
 import six
+import tempfile
 
 #: used during unit testing see ``pykern.pkunit.save_chdir``
 pkunit_prefix = None
 
 
+def atomic_write(filename, content, mode='w'):
```

Comment:
No need for mode.

Reply:
I think we should switch to "path" to indicate it might be a py.path. You can change write_text, too.

Make contents and content consistent, too.